ci/testnet-automation.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+echo --- downloading snap from build artifacts
+buildkite-agent artifact download "solana_*.snap" .
+
+# shellcheck disable=SC1091
+source ci/upload_ci_artifact.sh
+
+[[ -n $ITERATION_WAIT ]] || ITERATION_WAIT=300
+[[ -n $NUMBER_OF_NODES ]] || NUMBER_OF_NODES="10 25 50 100"
+
+launchTestnet() {
+  echo --- setup "$1" node test
```

Comment:
Let's `declare nodeCount=$1` then use `$nodeCount` in this function instead of `$1`, little more readable IMO